### ISSUE TITLE
Not showing more than 2000 files

### ISSUE THREAD

**Issue by treeol** [sallar/dropbox-fs#18]
Hi there

I'm facing the problem that it doesn't show me the whole amount of files.

In my case, I have more than 2000 files in the Dropbox-App-Folder. 

And dropbox-fs readDir is showing me only 2000 files. 

I opened it up as an issue because I didn't find anything documented about this.

Hope you can help me out here.

Sincerely

**Comment by jeremybarbet** [comment]
Yes that's normal, https://github.com/sallar/dropbox-fs/blob/master/src/index.js#L101, `filesListFolder ` only return the first ~2000 files. To be able to get the next X items, the `filesListFolderContinue` http://dropbox.github.io/dropbox-sdk-js/Dropbox.html#filesListFolderContinue__anchor and this function is missing from this package
